# HG changeset patch
# Branch conda
# Node ID a62de68281e3bc881cddc1d56646567dc543e3cd
# Parent  85c317f36cd8fc1962c1776a6cfb1df3572c518c
use conda layout in get_python_libs

diff -r 85c317f36cd8 -r a62de68281e3 lib-python/3/distutils/sysconfig_pypy.py
--- a/lib-python/3/distutils/sysconfig_pypy.py	Thu Jun 24 16:35:13 2021 +0300
+++ b/lib-python/3/distutils/sysconfig_pypy.py	Fri Jul 02 18:18:41 2021 +0300
@@ -47,14 +47,31 @@
     containing standard Python library modules; otherwise, return the
     directory for site-specific modules.
 
-    If 'prefix' is supplied, use it instead of sys.prefix or
-    sys.exec_prefix -- i.e., ignore 'plat_specific'.
+    If 'prefix' is supplied, use it instead of sys.base_prefix or
+    sys.base_exec_prefix -- i.e., ignore 'plat_specific'.
     """
     if prefix is None:
-        prefix = PREFIX
-    if standard_lib:
-        return os.path.join(prefix, "lib-python", sys.version[0])
-    return os.path.join(prefix, 'site-packages')
+        if standard_lib:
+            prefix = plat_specific and BASE_EXEC_PREFIX or BASE_PREFIX
+        else:
+            prefix = plat_specific and EXEC_PREFIX or PREFIX
+
+    if os.name == "posix":
+        libpython = os.path.join(prefix,
+                                 "lib", "python" + get_python_version())
+        if standard_lib:
+            return libpython
+        else:
+            return os.path.join(libpython, "site-packages")
+    elif os.name == "nt":
+        if standard_lib:
+            return os.path.join(prefix, "Lib")
+        else:
+            return os.path.join(prefix, "Lib", "site-packages")
+    else:
+        raise DistutilsPlatformError(
+            "I don't know where Python installs its library "
+            "on platform '%s'" % os.name)
 
 
 _config_vars = None
 
